profile/run_process_chunk.py

```
@@ -0,0 +1,300 @@
+                                + frames_np_chunk[idx] * (1 - binary_masks_chunk[idx])
+                            )
+
+                            if comp_frames_chunk[idx] is None:
+                                comp_frames_chunk[idx] = img
+                            else:
+                                comp_frames_chunk[idx] = (
+                                    comp_frames_chunk[idx].astype(np.float32) * 0.5
+                                    + img.astype(np.float32) * 0.5
+                                )
+
+        # 你用来中断 profiling 的断点，保留
```

Ruff (0.14.7)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)